### ISSUE TITLE
Crash when using Magnetic Ore Finder in Aura Pouch

### ISSUE THREAD

**Issue by Prunoideae** [ReikaKalseki/Reika_Mods_Issues#2308]
Magnetic Ore Finder can also be functional when in the CC's Aura Pouch. But it will crash the game if any ores on the screen were harvested.

Crash report:
https://pastebin.com/EQUsqitR

**Comment by ReikaKalseki** [comment]
No, this crash has nothing to do with the aura pouch; you have an ore block with null icons.

**Comment by Prunoideae** [comment]
I can't quite understand about this, the crash only happens when I mined any ores with Magnetic Ore finder in the pouch, for example the Magnetite Ore in RC, Certus Quartz Ore in AE2, it was reproduced for several times on my machine, and that's why I thought it was a bug, how do I supposed to make a ore block visible with null icons in the Ore Finder without have anything done on the source code?

**Comment by ReikaKalseki** [comment]
There is nothing you *can* do; that is a serious error in whatever mod is adding that ore.

**Comment by Prunoideae** [comment]
> There is nothing you _can_ do; that is a serious error in whatever mod is adding that ore.

I only have your mods, Tinker's Construct and Thaumcraft installed at the time, and that's why I think it's a bug and should be reported here.
Also, I can use the Magnetic Ore Finder without any crash outside the pouch, this ONLY occurs when I'm breaking ores rendered on the screen with pickaxe on hand and Finder in pouch, I don't know if there could be other things related, the problem even occurs at breaking placed ore in a superflat world, in a modpack where I created only with your mods: https://pastebin.com/FVE5Zv8C
It can't be wrong with other mods' ores, the only things bugged is somewhere in your code.

**Comment by ReikaKalseki** [comment]
None of my ores have null icons.

**Comment by Prunoideae** [comment]
Then that's the problem, why a magnetic finder will crash if none of your ores have null icons? It just  shouldn't be a crashed caused by null icons, but the mechanism to deal with block breaking of magnetic ore finder itself.
I made a further test using the pack with only your mod mentioned above, it even has nothing to do with the aura pouch itself, **just breaking the ore blocks with the magnetic ore finder in hand will produce a same crash**.
I know the bug reports said that it crashed because of a null icon is rendered, but if no ores have null icon, it just can't be a bug of modded ores have null icons.

![image](https://user-images.githubusercontent.com/24620047/74208596-359a7180-4cbf-11ea-9858-caa5b8370949.png)

Here is the mod list I used for reproducing the crash, I just don't know how can any of them produce the null icon ore.
